dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -965,8 +965,6 @@ vector<string> getOnUpdateTimestampColumns(string& schema, string& tableName, in
     boost::shared_ptr<CalpontSystemCatalog> csc = CalpontSystemCatalog::makeCalpontSystemCatalog(sessionID);
     csc->identity(execplan::CalpontSystemCatalog::FE);
     CalpontSystemCatalog::TableName aTableName;
-    boost::algorithm::to_lower(schema);
-    boost::algorithm::to_lower(tableName);
```

Comment:
I think we need the `if (lower_case_table_names)` block here.

Reply:
schema and tableName are guaranteed to have been converted before any call to getOnUpdateTimestampColumns